apps/sms/views/conversation/js/conversation.js

```diff
@@ -513,6 +514,9 @@ var ConversationView = {
     // TODO should we implement hooks to Navigation so that Threads could
     // get an event whenever the panel changes?
     Threads.currentId = args.id;
+    this.currentThreadId = args.id;
+    // TODO: Will need to replace with Conversation service method.
+    this.currentThread = Threads.active;
```

Comment:
I'm ok with the `activeThread`, and yes, `this.activeThread.id` should just work here.

We still need Threads before Conversation service ready, so you can see Threads.currentId still set here(and `Threads.active` is actually a getter that returning thread object once you set the valid currentId). So you can see `Threads.active` is still in used in somewhere and I just add a comment for it that it might be replace by Conversation service method.